devel/verify-vim-wsdl.rb

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env ruby
+
+require 'active_support/core_ext/enumerable'
+require 'active_support/inflector'
+require "optimist"
+require "pathname"
+require "rbvmomi"
+require "wsdl/parser"
+
+def parse_args(args)
+  opts = Optimist.options do
+    usage <<~HELP
+      Usage:
+      verify-vim-wsdl.rb [path to wsdl] [path to vmodl.db]
+    HELP
+
+    opt :fix, "Optionally fix the wsdl types in the vmodl.db", :type => :boolean, :default => false
+  end
+
+  Optimist.die("You must provide a wsdl file and a vmodl file") if args.count < 2
+
+  wsdl_path = Pathname.new(args.shift)
+  Optimist.die("You must pass a path to a wsdl file") if !wsdl_path.exist?
+
+  vmodl_path = Pathname.new(args.shift)
+  Optimist.die("You must pass a path to the vmodl.db file") if !vmodl_path.exist?
+
+  return wsdl_path, vmodl_path, opts
+end
+
+def in_directory(dir)
+  saved_dir = Dir.getwd
+  Dir.chdir(dir)
+  yield
+ensure
+  Dir.chdir(saved_dir)
+end
```

Comment:
@agrare Dir.chdir can take a block so I think this entire method can go away.

Reply:
:+1: good call, done